microsoft-edge/webview2/how-to/debug-visual-studio.md

```diff
@@ -35,59 +35,96 @@ To debug your code, first install and set up the script debugger tools in Visual
 <!-- ====================================================================== -->
 ## Install the JavaScript diagnostics component
 
-First, install the **JavaScript diagnostics** component in **Desktop development with C++**, as follows.
+First, install the **JavaScript diagnostics** component in the **Desktop development with C++** workload of Visual Studio, as follows.
+
+
+<!-- ------------------------------ -->
+#### If you're installing Visual Studio now
+
+1. To work with the WebView2 samples, download Visual Studio 2019 (version 16.4 Preview 2 or later), from [Downloads](https://visualstudio.microsoft.com/downloads/), and then run the downloaded installer file.  To work with your own WebView2 app, you might want to do these steps for Visual Studio 2022 as well.
```

Comment:
We should just ask them to use Visual Studio 2022. Don't need to ask them to have both versions